layouts/joomla/toolbar/modal.php

```diff
@@ -32,12 +32,14 @@
 ?>
 
 <!-- Render the button -->
-<joomla-toolbar-button<?php echo $id; ?> onclick="document.getElementById('modal_<?php echo $selector; ?>').open()" class="<?php echo $class; ?>" data-toggle="modal">
+<joomla-toolbar-button<?php echo $id; ?> onclick="document.getElementById('modal_<?php echo $selector; ?>').open();
+	$(document.getElementById('modal_<?php echo $selector; ?>')).appendTo('body');"
```

Comment:
```suggestion
	document.body.appendChild(document.getElementById('modal_<?php echo $selector; ?>'));"
```

Not at a machine with a setup but I think this should remove the jQuery dependency from this.